3ds Max/Max2Babylon/Exporter/BabylonExporter.cs

```diff
@@ -88,8 +89,103 @@ public void CheckCancelled()
                 throw new OperationCanceledException();
             }
         }
+
+        private void IsMeshCollapsable(IINode node, AnimationGroupList animationGroupList,ref List<IINode> collapsabeNodes)
```

Comment:
We're using flatten in one file, and collapse in another. we should probably stick to one term for this feature.

Reply:
Yes, I'm a Maya user, I'm used to name this feature collapse , in max is flatten
I replaced everything with flatten, flattenable etc.